src/app/queries/helpers.ts

```diff
@@ -27,6 +37,34 @@ export const useMockableQuery = <TResult = unknown, TError = unknown>(
     queryFn: process.env.DATA_SOURCE !== 'mock' ? config.queryFn : () => mockPromise(mockData),
   });
 
+// export const useMockableMutation = <
+//   TResult = unknown,
+//   TError = unknown,
+//   TVariables = unknown,
+//   TSnapshot = unknown
+// >(
+//   // mutationFn: MutationFunction<TResult, TVariables>,
+//   config: MutationConfig<TResult, TError, TVariables, TSnapshot>
+// ): MutationResultPair<TResult, TError, TVariables, TSnapshot> =>
+//   useMutation<TResult, TError, TVariables, TSnapshot>(
+//     process.env.DATA_SOURCE !== 'mock'
+//       ? mutationFn
+//       : (): any => window.alert('Sorry, this feature is not available in mock mode.'),
+//     config
+//   );
+
+//   {
+//   // ...config,
+//   // onMutate:()=> console.log('help')
+//   // mutationFn:()=> console.log('help')
+//     // process.env.DATA_SOURCE !== 'mock'
+//     //   ? config.onMutate
+//     //   : () => window.alert('Sorry, this feature is not available in mock mode.'),
+// }
+// );
+// export const getClusterApiUrl = (relativePath: string): string =>
+//   `${VIRT_META.clusterApi}/apis/virt.konveyor.io/v1alpha1/namespaces/openshift-migration${relativePath}`;
+
 export const getApiUrl = (relativePath: string): string =>
```

Comment:
Let's rename this to `getInventoryApiUrl` to distinguish it from `getClusterApiUrl`